superset-frontend/src/components/ListView/ListView.tsx

```diff
@@ -116,124 +118,117 @@ const ListView: FunctionComponent<Props> = ({
+            rows={rows}
+            loading={loading}
+          />
+        </div>
+        <div className="footer">
+          <Row>
+            <Col>
+              <div className="form-actions-container">
+                <div className="btn-group">
+                  {bulkActions.length > 0 && (
+                    <DropdownButton
+                      id="bulk-actions"
+                      bsSize="small"
+                      bsStyle="default"
+                      noCaret
+                      title={
+                        <>
+                          {t('Actions')} <span className="caret" />
+                        </>
+                      }
+                    >
+                      {bulkActions.map(action => (
                         // @ts-ignore
-                        onSelect={(selectedRows: typeof selectedFlatRows) => {
-                          action.onSelect(
-                            selectedRows.map((r: any) => r.original),
-                          );
-                        }}
-                      >
-                        {action.name}
-                      </MenuItem>
-                    ))}
-                  </DropdownButton>
-                )}
+                        <MenuItem
+                          key={action.key}
+                          eventKey={selectedFlatRows}
+                          // @ts-ignore
+                          onSelect={(selectedRows: typeof selectedFlatRows) => {
+                            action.onSelect(
+                              selectedRows.map((r: any) => r.original),
+                            );
+                          }}
+                        >
+                          {action.name}
+                        </MenuItem>
+                      ))}
+                    </DropdownButton>
+                  )}
+                </div>
               </div>
-            </div>
-          </Col>
-          <Col md={8} className="text-center">
-            <Pagination
-              totalPages={pageCount || 0}
-              currentPage={pageCount ? pageIndex + 1 : 0}
-              onChange={(p: number) => gotoPage(p - 1)}
-              hideFirstAndLastPageLinks
-            />
-          </Col>
-          <Col md={2}>
-            <span className="pull-right">
-              {t('showing')}{' '}
-              <strong>
-                {pageSize * pageIndex + (rows.length && 1)}-
-                {pageSize * pageIndex + rows.length}
-              </strong>{' '}
-              {t('of')} <strong>{count}</strong>
-            </span>
-          </Col>
-        </Row>
+            </Col>
+
+            <Col>
+              <span className="row-count-container">
+                {t('showing')}{' '}
+                <strong>
+                  {pageSize * pageIndex + (rows.length && 1)}-
+                  {pageSize * pageIndex + rows.length}
+                </strong>{' '}
+                {t('of')} <strong>{count}</strong>
```

Comment:
these translation strings won't work at all. I think using string replacement is required instead, details are here: https://github.com/apache-superset/superset-ui/tree/master/packages/superset-ui-translation

although, now that i look closer, it seems like these are just indentation changes. so i guess it's up to you if you want to fix them or not. not sure how they snuck past in the first place :/ if it's not in scope to fix them here, let me know and i can clean up afterwards

Reply:
@etr2460 I wrote this before I really understood how translation strings worked. Any idea how to get the html (`<strong>`) interspersed in the translation string?

Reply:
ah, it's not possible currently. I know Max was trying to figure out a way to hack around it, but I don't think we ever came up with a solution.

If the bold styling is necessary here, then i'd say remove the `t` strings entirely. If not, then it might be better to remove the styling for now to support better translations

Reply:
removed the `t` strings. I've also asked design to look into it with translation in mind.